### PR TITLE
[BACKLOG-5181] - Updated I18N files.

### PR DESCRIPTION
**Pull request by wseyler**
@dkincade for your review.

### CODE REVIEW

**Comment by dkincade** [comment]
Please remove the 3 new config files.

File: core/src/org/pentaho/platform/engine/core/messages/messages_de.properties [hunk 1]
```diff
@@ -70,7 +72,7 @@ PentahoSystem.USER_INITIALIZATION_SYSTEM_SETTINGS_FAILED=Fehler beim Lesen der K
 PentahoSystem.USER_SYSTEM_TITLE=Pentaho Business Intelligence-Plattform
 PentahoSystem.ERROR_0029_OUTPUT_HANDLER_NOT_SPECIFIED=Der Typ des Ausgabe-Handler wird in der Referenz nicht angegeben: {0}
 
-SYSTEMSETTINGS.CODE_LOG_NAME=SYSTEM-EINSTELLUNGEN
+SYSTEMSETTINGS.CODE_LOG_NAME=SYSTEM-SETTINGS
```
Comment (lucboudreau) [inline core/src/org/pentaho/platform/engine/core/messages/messages_de.properties]: This created a regression.

File: core/src/org/pentaho/platform/engine/core/messages/messages_fr.properties [hunk 2]
```diff
@@ -45,43 +45,45 @@ PentahoSystem.ERROR_0013_COULD_NOT_CREATE_OBEJCT=Impossible de cr\u00e9er l'obje
...
+PentahoSystem.USER_INITIALIZATION_SYSTEM_SETTINGS_FAILED=La lecture des informations de configuration du syst\u00e8me a \u00e9chou\u00e9. L'emplacement de cette configuration est sp\u00e9cifi\u00e9 soit dans le fichier web.xml sous le param\u00e8tre contextuel \u00ab pentaho-system-cfg \u00bb, soit avec l'argument JVM \u00ab {0} \u00bb (par exemple -D{0}=C:/system/pentaho.xml).
 PentahoSystem.USER_SYSTEM_TITLE=Pentaho Business Intelligence Platform
-PentahoSystem.ERROR_0029_OUTPUT_HANDLER_NOT_SPECIFIED=Type de programme de traitement des donn\u00e9es de sortie non sp\u00e9cifi\u00e9 dans la r\u00e9f\u00e9rence\u0020: {0}
+PentahoSystem.ERROR_0029_OUTPUT_HANDLER_NOT_SPECIFIED=Type de programme de traitement des donn\u00e9es de sortie non sp\u00e9cifi\u00e9 dans la r\u00e9f\u00e9rence : {0}
 
-SYSTEMSETTINGS.CODE_LOG_NAME=PARAM\u00c8TRES DU SYST\u00c8ME
+SYSTEMSETTINGS.CODE_LOG_NAME=SYSTEM-SETTINGS
```
Comment (lucboudreau) [inline core/src/org/pentaho/platform/engine/core/messages/messages_fr.properties]: This created a regression.

File: core/src/org/pentaho/platform/engine/security/messages/ldapuserdetailsservice_messages_ja.properties [hunk 3]
```diff
@@ -1,4 +1,4 @@
-LdapUserDetailsService.ERROR_0001_NAMING_EXCEPTION=\u4f8b\u5916\u304c\u767a\u751f\u3057\u307e\u3057\u305f\u3002
-LdapUserDetailsService.ERROR_0002_USERSEARCH_NOT_SPECIFIED=userSearch \u306f\u8a2d\u5b9a\u304c\u5fc5\u8981\u3067\u3059\u3002
-LdapUserDetailsService.ERROR_0003_POPULATOR_NOT_SPECIFIED=populator \u306f\u8a2d\u5b9a\u304c\u5fc5\u8981\u3067\u3059\u3002
-LdapUserDetailsService.ERROR_0004_USERDETAILSMAPPER_NOT_SPECIFIED=userDetailsMapper \u306f\u8a2d\u5b9a\u304c\u5fc5\u8981\u3067\u3059\u3002
+LdapUserDetailsService.ERROR_0001_NAMING_EXCEPTION=an exception occurred
+LdapUserDetailsService.ERROR_0002_USERSEARCH_NOT_SPECIFIED=userSearch must be set
+LdapUserDetailsService.ERROR_0003_POPULATOR_NOT_SPECIFIED=populator must be set
+LdapUserDetailsService.ERROR_0004_USERDETAILSMAPPER_NOT_SPECIFIED=userDetailsMapper must be set
```
Comment (lucboudreau) [inline core/src/org/pentaho/platform/engine/security/messages/ldapuserdetailsservice_messages_ja.properties]: All of these created regressions. It should be Japanese.

File: core/src/org/pentaho/platform/engine/security/messages/messages_ja.properties [hunk 4]
```diff
@@ -27,23 +27,23 @@ PentahoAclEntry.USER_UPDATE=\u66f4\u65b0
...
 
-AclPublisher.ERROR_0001_DEFAULT_ACL_REQUIRES_USER_OR_ROLE=\u30c7\u30d5\u30a9\u30eb\u30c8\u306e ACL \u306f\u6307\u5b9a\u3055\u308c\u305f\u30e6\u30fc\u30b6\u30fc\u3082\u3057\u304f\u306f\u30ed\u30fc\u30eb\u304c\u5fc5\u9808\u3067\u3059\u3002
-AclPublisher.ERROR_0002_DEFAULT_ACL_HAS_BOTH=\u30c7\u30d5\u30a9\u30eb\u30c8\u306e ACL \u306f\u6307\u5b9a\u3055\u308c\u305f\u30e6\u30fc\u30b6\u30fc\u3082\u3057\u304f\u306f\u30ed\u30fc\u30eb\u304c\u5fc5\u9808\u3067\u3059\u304c\u3001\u4e21\u65b9\u3068\u3082\u6307\u5b9a\u3055\u308c\u3066\u3044\u307e\u305b\u3093\u3002
-DefaultPentahoPasswordEncoder.ERROR_0001_RAWPASS_CANNOT_BE_NULL=raw \u30d1\u30b9\u306f\u5fc5\u9808\u3067\u3059\u3002
-DefaultPentahoPasswordEncoder.ERROR_0002_ENCPASS_CANNOT_BE_NULL=enc \u30d1\u30b9\u306f\u5fc5\u9808\u3067\u3059\u3002
+AclPublisher.ERROR_0001_DEFAULT_ACL_REQUIRES_USER_OR_ROLE=Default ACL must have either a user or a role specified
+AclPublisher.ERROR_0002_DEFAULT_ACL_HAS_BOTH=Default ACL must have either a user or a role specified, but not both
+DefaultPentahoPasswordEncoder.ERROR_0001_RAWPASS_CANNOT_BE_NULL=rawPass cannot be null
+DefaultPentahoPasswordEncoder.ERROR_0002_ENCPASS_CANNOT_BE_NULL=encPass cannot be null
```
Comment (lucboudreau) [inline core/src/org/pentaho/platform/engine/security/messages/messages_ja.properties]: All of these should be in Japanese.

**Pull request merged**